tests/unit/modules/network/eos/test_eos_route_maps.py

```diff
@@ -0,0 +1,481 @@
+#
+# (c) 2019, Ansible by Red Hat, inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from ansible_collections.arista.eos.tests.unit.compat.mock import patch
+from ansible_collections.arista.eos.plugins.modules import eos_route_maps
+from ansible_collections.arista.eos.tests.unit.modules.utils import (
+    set_module_args,
+)
+from .eos_module import TestEosModule, load_fixture
+
+
+class TestEosRoute_MapsModule(TestEosModule):
+    module = eos_route_maps
+
+    def setUp(self):
+        super(TestEosRoute_MapsModule, self).setUp()
+
+        self.mock_get_resource_connection_config = patch(
+            "ansible_collections.ansible.netcommon.plugins.module_utils.network.common.rm_base.resource_module_base.get_resource_connection"
+        )
+        self.get_resource_connection_config = (
+            self.mock_get_resource_connection_config.start()
+        )
+
+        self.mock_execute_show_command = patch(
+            "ansible_collections.arista.eos.plugins.module_utils.network.eos.facts.route_maps.route_maps.Route_mapsFacts.get_config"
+        )
+        self.execute_show_command = self.mock_execute_show_command.start()
+
+    def tearDown(self):
+        super(TestEosRoute_MapsModule, self).tearDown()
+        self.mock_get_resource_connection_config.stop()
+        self.mock_execute_show_command.stop()
+
+    def load_fixtures(self, commands=None, transport="cli", filename=None):
+        if filename is None:
+            filename = "eos_route_maps_config.cfg"
+
+        def load_from_file(*args, **kwargs):
+            output = load_fixture(filename)
+            return output
+
+        self.execute_show_command.side_effect = load_from_file
+
+    def test_eos_route_maps_merged_idempotent(self):
```

Comment:
wanted to check if you covered all the params supported under route_maps RM via unit TC, as now we decided to tc of all module params which can give best possible coverage.

Reply:
The coverage for argspecs for this module is > 80%. Having said that, I have submitted a separate issue to increase the test coverage in eos modules. I will be working on this module along with other modules, to close the issue. (https://github.com/ansible-collections/arista.eos/issues/204)